WanderLost/WanderLost/Client/Components/Merchant.razor.css

```diff
@@ -32,4 +32,8 @@
 
 .merchant__deck ::deep .merchant__card--tetriary {
     transform: rotate(6deg);
-}
\ No newline at end of file
+}
+
+.button_suggest {
```

Comment:
for element you use `__` as a separator :) `_` goes for modifier name

Reply:
Sure, will change